code/modules/client/client procs.dm

```diff
@@ -258,8 +260,11 @@
 
 	if(connection != "seeker")					//Invalid connection type.
 		return null
-	if(byond_version < MIN_CLIENT_VERSION)		//Out of date client.
+	if(byond_version < MIN_CLIENT_VERSION) // Too out of date to play at all, force update.
+		to_chat(src,"<span class='userdanger'>Your BYOND client (v: [byond_version]) is too far out of date to to play. Get the latest client from http://www.byond.com/")
```

Comment:
This message will never actually be visible; booting a client in /client/New disconnects them before the BYOND UI even shows up. Using `alert`, as the guest removal does below, doesn't work either. Neither does `src << "message"`...

I'm not sure there's actually any way to send a client a message before disconnecting them here. You might actually have to let them connect, show them the message, and *then* disconnect them shortly after, if that can even be done reliably with how long it takes to load in sometimes.